server/router/driver.ts

```diff
@@ -12,6 +12,52 @@ import { UserType } from '../models/subscription';
 const router = express.Router();
 const tableName = 'Drivers';
 
+// Get all drivers
+router.get('/', validateUser('Admin'), (req, res) => {
+  db.getAll(res, Driver, tableName);
+});
+
+// Get all available drivers at a specific date and time
+router.get('/available', validateUser('User'), (req, res) => {
+  const { date, startTime: reqStartTime, endTime: reqEndTime } = req.query;
+  const reqDay = moment(date as string).day();
+
+  if ((reqStartTime as string) >= (reqEndTime as string)) {
+    res.status(400).send({ err: 'startTime must precede endTime' });
+  }
+
+  db.getAll(res, Driver, tableName, (doc: DriverType[]) => {
```

Comment:
No. Dynamoose condition is not able to check if a startTime is greater than another time because not all drivers have startTime on that particular day.